src/pages/home/ReportScreen.js

```diff
@@ -487,6 +487,7 @@ function ReportScreen({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    const reportIDParams = lodashGet(route.params, 'reportID');
```

Comment:
NAB: reportIDFromParams would make more sense.

Reply:
You can fix in another PR if you want

Reply:
Updated